src/internal_events/azure_queue.rs

```diff
@@ -0,0 +1,95 @@
+use metrics::counter;
+use vector_lib::internal_event::{error_stage, error_type, InternalEvent};
+#[cfg(feature = "sources-azure_blob")]
+pub use azure_blob::*;
```

Comment:
that is for if you want to import events from azure_blob mod inside of this file instead of doing internal_events::azure_blobs just doing internal_events::{some_event} in its usages. Can delete and keep it as usual, no big arguments there.